argo_entrypoint.sh

```diff
@@ -43,7 +43,9 @@ done; \
 echo "DEBUG: Gitea API available";
 
 # Overwrite content before import
-cp -rv "/shared/"* "${CF_IMPORT_CONTENT}/" && echo "INFO: overwriting /content success" || echo "ERROR: overwriting /content failed"
+if [[ -e "/shared" ]]; then
+  cp -rv "/shared/"* "${CF_IMPORT_CONTENT}/" && echo "INFO: overwriting /content success"
+fi
```

Comment:
```suggestion
else
  echo "INFO: /shared does not exist"
fi
```

Reply:
Actually /shared and /content directories are created using docker file so they definitely exists . we are actually checking /shared directory has any files or folders in it else it skips overwritting /content

Reply:
I based the message on the shell code in the PR:
```bash
[[ -e "/shared" ]]
```

That tests if /shared exists. That does not test if it has any contents